cmd/machine.go

```diff
@@ -32,10 +32,10 @@ var machineCmd = &cobra.Command{
 			os.Exit(1)
 		}
 		defer zlog.Sync() // nolint
-		logger.V(0).Info("machine actions")
+		//logger.V(0).Info("machine actions")
 
 		opts = append(opts, grpc.WithInsecure())
-		conn, err := grpc.Dial("localhost:50051", opts...)
+		conn, err := grpc.Dial("localhost:"+port, opts...)
```

Comment:
Is this to connect to the local pbnj daemon?

Reply:
yes, will need to come back around and make `localhost` a flag too to allow for interaction with remote instances.